systems_cross_building/cross_building.rst

```diff
@@ -701,8 +701,8 @@ You can use a profile like the following to cross-build your packages for ``iOS`
 
     **ARM**
 
-    - https://developer.arm.com/docs/dui0773/latest/compiling-c-and-c-code/specifying-a-target-architecture-processor-and-instruction-set
-    - https://developer.arm.com/docs/dui0472/latest/compiler-command-line-options
+    - https://developer.arm.com/documentation/dui0773/latest/compiling-c-and-c-code/specifying-a-target-architecture-processor-and-instruction-set
```

Comment:
Same here, the one that works in my browser is: https://developer.arm.com/documentation/dui0773/j/compiling-c-and-c---code/specifying-a-target-architecture--processor--and-instruction-set